### ISSUE TITLE
Could not load type 'ArchiSteamFarm.Plugins.Interfaces.IBotCommand'

### ISSUE THREAD

**Issue by woctezuma** [Ryzhehvost/BoosterCreator#11]
Error:
```
2022-03-03 00:40:41|dotnet-14196|ERROR|ASF|InitPlugins() Échec dû à une erreur : BoosterCreator, Version=1.2.1.0, Culture=neutral, PublicKeyToken=null
2022-03-03 00:40:41|dotnet-14196|ERROR|ASF|InitPlugins() System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load type 'ArchiSteamFarm.Plugins.Interfaces.IBotCommand' from assembly 'ArchiSteamFarm, Version=5.2.3.5, Culture=neutral, PublicKeyToken=504f46588cc351a9'.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at ArchiSteamFarm.Plugins.PluginsCore.InitPlugins()
System.TypeLoadException: Could not load type 'ArchiSteamFarm.Plugins.Interfaces.IBotCommand' from assembly 'ArchiSteamFarm, Version=5.2.3.5, Culture=neutral, PublicKeyToken=504f46588cc351a9'.
```

Possible Explanation:
>[[Deprecation](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Deprecation)] ASF plugins that implement IBotCommand interface or use Bot.HasAccess(), Commands.Response() functionality will need slight code changes in regards to changes done as part https://github.com/JustArchiNET/ArchiSteamFarm/pull/2501. We tried our best to keep them working as part of this release - the deprecated functionality will be removed in the next release cycle.
>
> https://github.com/JustArchiNET/ArchiSteamFarm/releases/tag/5.2.2.4

Useful references:
- https://github.com/JustArchiNET/ArchiSteamFarm/pull/2501
- https://github.com/JustArchiNET/ArchiSteamFarm/issues/2500

**Comment by woctezuma** [comment]
I can see several spots where the issue could come from:

https://github.com/Ryzhehvost/BoosterCreator/blob/996f06c71f4c12baec68d07638d08b59295fb13e/BoosterCreator/Commands.cs#L31

**Comment by Ryzhehvost** [comment]
Yes, that's expected, most if not all my plugins will not work with new ASF version. Of course I'm willing to update plugins, but due to war in my country I can't tell you when (or if) this will happen.

**Comment by woctezuma** [comment]
Sure.  ❤️ I guessed so, but still posted the issue so that it would be documented. I may try to have a look at the code.

**Comment by woctezuma** [comment]
Fixed by #12.
